clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -589,7 +589,8 @@
 
     // currentThread holds the threadId of the current thread accessing KafkaConsumer
     // and is used to prevent multi-threaded access
-    private final AtomicLong currentThread = new AtomicLong(NO_CURRENT_THREAD);
+    private final AtomicLong currentThreadId = new AtomicLong(NO_CURRENT_THREAD);
```

Comment:
Is there a need to keep the thread id around if the Thread is already being referenced?

Reply:
fixed